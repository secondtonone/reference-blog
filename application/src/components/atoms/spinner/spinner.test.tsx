import renderer from 'react-test-renderer';
import Spinner from '.';

describe('Match snapshot Spinner', () => {
  it('default', () => {
    const tree = renderer.create(<Spinner />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
