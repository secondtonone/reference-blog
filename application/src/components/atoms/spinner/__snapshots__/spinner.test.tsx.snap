// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match snapshot Spinner default 1`] = `
<div
  className="spinner__SpinStyled-sc-15rva1s-0 byYMuV"
  data-test="spinner"
>
  <svg
    fill="none"
    height={65}
    viewBox="0 0 66 66"
    width={65}
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      cx="33"
      cy="33"
      fill="none"
      id="spinner"
      r="30"
      stroke="url(#gradient_1)"
      strokeLinecap="round"
      strokeWidth="6"
    />
    <defs>
      <linearGradient
        id="gradient_1"
        x1="0%"
        x2="0%"
        y1="0%"
        y2="100%"
      >
        <stop
          offset="0%"
          stopColor="#FF642D"
          stopOpacity="0"
        />
        <stop
          offset="100%"
          stopColor="#FF642D"
        />
      </linearGradient>
      <linearGradient
        id="gradient_2"
        x1="0%"
        x2="0%"
        y1="0%"
        y2="100%"
      >
        <stop
          offset="0%"
          stopColor="#FFC600"
          stopOpacity="0"
        />
        <stop
          offset="100%"
          stopColor="#FFC600"
        />
      </linearGradient>
      <linearGradient
        id="gradient_3"
        x1="0%"
        x2="0%"
        y1="0%"
        y2="100%"
      >
        <stop
          offset="0%"
          stopColor="#45E0A8"
          stopOpacity="0"
        />
        <stop
          offset="100%"
          stopColor="#45E0A8"
        />
      </linearGradient>
      <linearGradient
        id="gradient_4"
        x1="0%"
        x2="0%"
        y1="0%"
        y2="100%"
      >
        <stop
          offset="0%"
          stopColor="#6EDBFF"
          stopOpacity="0"
        />
        <stop
          offset="100%"
          stopColor="#6EDBFF"
        />
      </linearGradient>
      <linearGradient
        id="gradient_5"
        x1="0%"
        x2="0%"
        y1="0%"
        y2="100%"
      >
        <stop
          offset="0%"
          stopColor="#421983"
          stopOpacity="0"
        />
        <stop
          offset="100%"
          stopColor="#421983"
        />
      </linearGradient>
    </defs>
  </svg>
</div>
`;
